bin/ch_misc.c

```diff
@@ -450,18 +465,19 @@ noreturn void msg_fatal(const char *file, int line, int errno_,
 void msgv(enum log_level level, const char *file, int line, int errno_,
           const char *fmt, va_list ap)
 {
+   char *message = "";
```

Comment:
Initialize this to `NULL` (or not at all) so it doesn't look like you're trying to write to a static string.